test/parallel/test-child-process-fork-net-server.js

```diff
@@ -21,6 +21,10 @@
 
 'use strict';
 const common = require('../common');
+// TODO
```

Comment:
TODO what? Also, can you add your name to the comment (`TODO(username): description of comment`). Same comment applies elsewhere in this PR.